app/src/main/java/com/project/chat/MessageAdapter.java

```diff
@@ -35,9 +35,22 @@ public MessageAdapter(List<MessageItem> messagesList, String currentGroupName) {
     }
 
     public class MessageViewHolder extends RecyclerView.ViewHolder {
-
-
-
+        // my message
+        public TextView myName, myMessage, myMessageTime;
+        // other message
+        public TextView otherName, otherMessage, otherMessageTime;
+        public ImageView otherProfileImage;
+        public MessageViewHolder(@NonNull @NotNull View itemView) {
```

Comment:
Can you space between variables and functions from next time?
Anyway, I'm gonna commit now